components/forecast/AvalancheTab.tsx

```diff
@@ -53,88 +52,66 @@ export const AvalancheTab: React.FunctionComponent<AvalancheTabProps> = React.me
 
   const elevationBandNames: ElevationBandNames = zone.config.elevation_band_names;
 
+  const [textColor] = useToken('colors', ['darkText']);
+  const renderHTMLProps: Partial<RenderHTMLProps> = {
+    contentWidth: windowWidth,
+    defaultTextProps: {
+      style: {
+        fontSize: 16,
+        color: textColor,
+      },
+    },
+  };
+
   return (
-    <VStack space="4" bgColor={'#f0f2f5'}>
-      <Card paddingTop={2} borderRadius={0} borderColor="white" header={<Heading>Avalanche Forecast</Heading>}>
+    <VStack space="2" bgColor={'#f0f2f5'}>
+      <Card marginTop={2} borderRadius={0} borderColor="white" header={<Heading>Avalanche Forecast</Heading>}>
```

Comment:
Probably a better question for Charlotte & the designers, but what is this header doing for us? We're inside of an avalanche forecast tab, which someone clicked on specifically, and we're already showing the zone at the top of the screen - do we really need to spend 10% of our vertical real estate above the fold on a big header that delivers little info?

Reply:
I think this is a discussion to have in Figma, or while user-testing - I don't have the answer you seek. 

But also, the designers have explicitly said that the Avalanche Forecast page is a WIP, so I wouldn't get too worried either way. We can iterate.

Reply:
Tracked in https://github.com/stevekuznetsov/avalanche-forecast/issues/45